server/apps/archive/archive_ingest.py

```diff
@@ -110,7 +113,8 @@ def create(self, docs, **kwargs):
                 superdesk.get_resource_service(ARCHIVE).post([dest_doc])
                 insert_into_versions(dest_doc.get('guid'))
 
-                push_notification('item:fetch', item=str(ingest_doc.get('_id')))
+        if(kwargs.get('notify', True)):
```

Comment:
It is all single repo problem. Before that I would never us js in python :bowtie: